apitools/scripts/oauth2l.py

```diff
@@ -173,20 +167,20 @@ def _ValidateToken(access_token):
     return bool(_GetTokenScopes(access_token))
 
 
-def FetchCredentials(scopes, client_info=None, credentials_filename=None):
+def FetchCredentials(args, client_info=None, credentials_filename=None):
```

Comment:
actually, not done -- but made the function private instead. (nothing else depends on this file, so i'm not too worried about wanting to change the API later if it needs to.)